examples/nidcpower_source_dc_voltage/_nidcpower_helpers.py

```diff
@@ -7,21 +7,16 @@
 
 import ni_measurementlink_service as nims
 
-USE_SIMULATION = True
-"""
-To use a physical NI SMU instrument, set this to False or specify
---no-use-simulation on the command line.
-"""
-
 
 def create_session(
     session_info: nims.session_management.SessionInformation,
+    use_simulation: bool,
```

Comment:
`teststand_fixture.py` is still using the old signature.

https://github.com/ni/measurementlink-python/blob/main/examples/nidcpower_source_dc_voltage/teststand_fixture.py#L63
```
                _ = create_session(
                    session_info,
                    grpc_device_channel,
                    initialization_behavior=nidcpower.SessionInitializationBehavior.INITIALIZE_SERVER_SESSION,
                )
```

Reply:
I have reverted the changes specific to `nidcpower` and `nivisa_dmm` measurement. Will take this up as part of work item 2473562.